### PR TITLE
User Avatar - Multiple Users component

### PR DESCRIPTION
**Pull request by wes337**
New designs for the UserAvatar/MultipleUsers component: https://www.figma.com/file/CS01VVLR7ArQl0afYFkNj3/Web-App?node-id=4273%3A0

multiuser

Part of #1324

### CODE REVIEW

**Review by HarrisonMayotte — APPROVED** [review]

**Review by HarrisonMayotte — APPROVED** [review]

**Comment by goldyydev** [comment]
Wouldn't this fuck the desktop view up?

**Review by goldyydev — CHANGES_REQUESTED** [review]
Pretty sure this would fuck the desktop view up

**Comment by ajmnz** [comment]
It doesn't for me. Why would that mess desktop view?

Screenshot 2021-03-20 at 16 45 09

**Comment by ajmnz** [comment]
It's an intended modification, not a mobile-exclusive feature

**Review by goldyydev — APPROVED** [review]
Ah, thought it was supposed to be mobile exclusive

**Comment by wes337** [comment]
Actually, should this take a size prop (default to xs)?

**Comment by goldyydev** [comment]
Probably

File: kibbeh/src/ui/UserAvatar/MultipleUsers.tsx [hunk 1]
```diff
@@ -4,24 +4,26 @@ import { SingleUser } from "./SingleUser";
...
-    <div
-      className="flex relative"
-      style={{ width: `${srcArray.length * 7 + 18}px` }}
-    >
+    <div className={className}>
       {srcArray.slice(0, 3).map((s, i) => (
         <span
           key={s + i}
-          className="absolute box-content bg-primary-800 rounded-full border-primary-800 top-1/2 transform -translate-y-1/2"
+          className=" rounded-full bg-primary-800 border-primary-800"
```
Comment (goldyydev) [inline kibbeh/src/ui/UserAvatar/MultipleUsers.tsx]:
```suggestion
          className="rounded-full bg-primary-800 border-primary-800"
```

**Review by goldyydev — CHANGES_REQUESTED** [review]

**Review by goldyydev — APPROVED** [review]

**Pull request merged**